### PR TITLE
implented GET /api/reviews and tested the functions

### PR DESCRIPTION
**Pull request by LSB616**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ const db = require("../db/connection");
 const app = require("../app");
 const { string } = require("pg-format");
```
Comment (Jodiech) [inline __tests__/app.test.js:4]: You can delete line 4. This may have auto imported.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -32,4 +36,35 @@ describe('GET /api/categories', () => {
...
+          comment_count: expect.any(String)
+      }
+      reviews.forEach(review => {
+      expect(review).toEqual(expect.objectContaining(expected))
+      });
+      expect(reviews.length).toBe(24);
+      const countTest = reviews.filter(review => review.review_id === 1)
+      expect(countTest[0].comment_count).toBe('3');
+      expect(reviews).toBeSortedBy('created_at', {
+        descending: true
```
Comment (Jodiech) [inline __tests__/app.test.js:67]: Could you explain what you are doing here? Do we need this complex logic here?

**Review by Jodiech — COMMENTED** [review]
Nice work so far Lewis 👍 please amend your code, happy for you to merge once this is done :)

File: __tests__/app.test.js (hunk 2 above)
Comment (LSB616) [reply __tests__/app.test.js:67]: Sure, the first test is just testing my SQL to ensure that the required comment_count functionality from the trello post was working and the second test is using the Jest add-on to check that the required sorted by data function is working and it is doing it in descending order.

**Pull request merged**